ansible_base/activitystream/models/entry.py

```diff
@@ -65,3 +67,14 @@ def activity_stream_entries(self):
         A helper property that returns the activity stream entries for this object.
         """
         return Entry.objects.filter(content_type=ContentType.objects.get_for_model(self), object_id=self.pk).order_by('created')
+
+    def extra_related_fields(self, request):
+        content_type = ContentType.objects.get_for_model(self)
+        query_kwargs = {
+            'content_type': content_type.pk,
+            'object_id': self.pk,
+        }
+        activity_stream_url = reverse('activitystream-list') + '?' + urlencode(query_kwargs)
+        return {
```

Comment:
... `if "ansible_base.activity_stream" in settings.INSTALLED_APPS and isinstance(self, AuditableModel)`?

Reply:
This is on `AuditableModel`, not `CommonModel`, so I think we're fine here. The only way something would subclass `AuditableModel` is by having activity stream installed.